### ISSUE TITLE
Typo in Figure 0.4 - chapter 0 - page 8

### ISSUE THREAD

**Issue by potato16** [jeffgerickson/algorithms#31]
I think it should be a letter 'B' instead of 'D'
![image](https://user-images.githubusercontent.com/8458733/50631185-0373f780-0f76-11e9-81d0-2c5527ea8c0f.png)

**Comment by MarkJeronimus** [comment]
Also there should be a line intersecting E and C parallell to γ

**Comment by jeffgerickson** [comment]
I'm seeing a B there, not a D; see my attached screenshot.  Can you confirm that you're looking at the most recent version?

Notice that the A in your screenshot is missing its horizontal crossbar.  I suspect this is an issue your PDF viewer, and possibly with embedded fonts in the figure file.

<img width="1328" alt="algorithms-jeffe pdf page 26 of 448" src="https://user-images.githubusercontent.com/2807510/51083070-c8ab5580-1713-11e9-898c-4c4ba5fb7a44.png">

**Comment by echuber2** [comment]
I'm not seeing this issue on Linux with any of Evince, Okular, Firefox, or Chrome.

If it is an issue with broken font hints or Windows font rendering, the solution might be to convert text to shapes in Omnigraffle, or to use Fontforge to make a stripped version of the font with no hints just for use in figures.

**Comment by MarkJeronimus** [comment]
Ooh  a Mandelbrot avatar.

Linux renders fonts completely different from windows, so no surprise there.

I confirmed it's a hint problem by zooming in on the video. You clearly see that the horizontal bar of the A is moved up.

![Font kerning fail.gif](https://i.imgur.com/jVs4L3h.gifv)

**Comment by MarkJeronimus** [comment]
The copy markdown code function on imgur is broken. Try #2.

![Font kerning fail.gif](https://i.imgur.com/jVs4L3h.gif)

**Comment by jeffgerickson** [comment]
This appears to be a Windows/Adobe bug.  On my Mac, this **does not** happen in TeXshop, Preview, or Skim (which all use Apple's pdfkit under the hood) but it **does** happen in Acrobat and Acrobat Reader DC (which obviously don't).  Which explains why I never noticed it.

So I "just" have to do is figure out how to strip the hinting out of the PDF files that OmniGraffle exports.  There are a **lot** of figures where this could be a problem.

tl;dr: 🤦‍♂️

**Comment by jeffgerickson** [comment]
This _appears_ to be a problem with the (default) TrueType version of Bitstream Charter (Charter Italic; 14.0d2e1; 2017-11-28) included with Mac OS X (High Sierra, but the same version is included in Mojave).  There is an older free version of Charter available on the web (in both ttf and otf), which may be the one mathdesign is using, but I'm hesitant to use it because of reduced Unicode support.  Yuck.  Yuck.  Yuck.

**Comment by echuber2** [comment]
I suggested two other potential solutions above but I don't know how they'd figure into your workflow.

**Comment by jeffgerickson** [comment]
Eric: I didn't realize OmniGraffle had a "Convert Text to Shapes" operation.  Yes, it works (confirmed in Acrobat), but as you suggest, once the text is converted, it's impossible to edit as text.

It looks like the older free version of Charter Italic doesn't have the same hinting problems as Apple's version, but with exactly the same character repertoire.  (There may be some teeny typographical improvements between the free version and Apple's, but those are inaccessible anyway, because the body text already uses the free version via `\usepackage[charter]{mathdesign}`.)

See also the serifs on the crossbars of E and F, the different widths of the top and bottom of D and O and Q, and the 

![Apple Charter kerning fail](https://i.imgur.com/e12UHvm.png)

![Free Charter kerning](https://i.imgur.com/xaVTfyS.png)

Now that I have the non-broken font installed, I "just" need to re-export every OmniGraffle figure with Charter text as a fresh PDF. Whee!

**Comment by jeffgerickson** [comment]
I need to replace/double-check the following figures: 1.10, 1.11, 1.13, 1.14, 1.15, 1.21, Ex.1.42, 3.1, 3.2, 5.4, 5.5, 5.9, 5.10, 5.11, Ex.5.14, Ex.5.15, 5.17, 6.4, 6.5, 6.6, 6.8, 6.13, 6.14, 6.17, 7.2, 8.1, 8.3, 8.4, 8.6, 8.17, Ex.8.16, Ex.8.17, 9.1, 9.3, 10.2, 10.3, 10.5, 10.6, 10.7, 10.8, 10.9, 10.12, 10.13, 11.1, 11.2, 11.3, 11.4, 11.5, 11.6, 11.8, 12.1, 12.2, 12.5, [SAT reduction], 12.6, 12.7, 12.8, 12.10, 12.11, [3Color reduction], 12.12, 12.13, 12.15, [HamCycle reduction], 12.16, 12.17, 12.18, 12.19, 12.20

**Comment by jeffgerickson** [comment]
I believe I've fixed all occurrences of this bug, but if I've missed one, please open an new issue.
